web-bundle/src/main/java/com/graphhopper/resources/RouteResource.java

```diff
@@ -151,10 +148,11 @@ public Response doGet(
 
         GHResponse ghResponse = graphHopper.route(request);
 
-        float took = sw.stop().getSeconds();
+        long took = sw.stop().getNanos() / 1000;
         String infoStr = httpReq.getRemoteAddr() + " " + httpReq.getLocale() + " " + httpReq.getHeader("User-Agent");
+        // todonow: is there any reason we should keep logging vehicle/weighting? maybe to keep track of the number of 'legacy' requests or something?
```

Comment:
Probably not?

Reply:
I would indeed probably keep it until 2.0?

Reply:
For GET its already contained in httpReq.getQueryString(). Same for `algoStr`, but I added it explicitly now.